### PR TITLE
Add stats from dicom tags - BIDS input

### PR DESCRIPTION
**Pull request by arnaudbore**
#41

### CODE REVIEW

File: scripts/dmriqc_dwi_protocol.py [hunk 1]
```diff
@@ -53,6 +63,18 @@ def main():
     if not len(args.bval) == len(args.bvec):
         parser.error("Not the same number of images in input.")
 
+    stats_tags_for_graph = []
+    if args.metadata:
+        if not len(args.metadata) == len(args.bval):
+            print('Number of metadata files: {}.\n'
```
Comment (GuillaumeTh) [inline scripts/dmriqc_dwi_protocol.py:69]: Please use parser.error

File: dmriqcpy/viz/utils.py [hunk 2]
```diff
@@ -32,11 +32,11 @@ def analyse_qa(stats_per_subjects, stats_across_subjects, column_names):
         for name in stats_per_subjects.index:
             if stats_per_subjects.at[name, metric] > mean + 2 * std or\
                stats_per_subjects.at[name, metric] < mean - 2 * std:
-                warning[metric].append(name[0])
```
Comment (GuillaumeTh) [inline dmriqcpy/viz/utils.py:35]: Why did you change this ?

Comment (arnaudbore) [reply dmriqcpy/viz/utils.py:35]: I think, it is easier to read.

Comment (arnaudbore) [reply dmriqcpy/viz/utils.py:35]: I used this so instead of reading the whole *.bval it use the path+baseline but without extension.

**Review by GuillaumeTh — APPROVED** [review]
LGTM ! GJ ! :beers:
